grails-app/controllers/org/pih/warehouse/api/PurchaseOrderApiController.groovy

```diff
@@ -152,10 +152,10 @@ class PurchaseOrderApiController {
                 order?.paymentMethodType?.name,
                 order?.paymentTerm?.name,
                 lineItemsSize,
-                order?.orderedOrderItems?.size() ?: 0,
-                order?.shippedOrderItems?.size() ?: 0,
-                order?.receivedOrderItems?.size() ?: 0,
-                order?.invoiceItems?.size() ?: 0,
+                it?.itemsOrdered ?: 0,
+                it?.itemsShipped ?: 0,
+                it?.itemsReceived ?: 0,
+                it?.itemsInvoiced ?: 0,
```

Comment:
Just a small suggestion even though it has been there for a long time, but maybe it'd be worth to change in line 117 from:
```groovy
def getOrdersCsv(List purchaseOrders)
```
to:
```groovy
def getOrdersCsv(List<OrderSummary> purchaseOrders)
```
so that the compiler understands what `itemsOrdered` etc is, instead of marking it as grey and saying: "_No candidates found for method call it?.itemsOrdered."_

Reply:
Yup, I want to do this plus stuff like changing `it` to named var, etc